tools/report.sh

```diff
@@ -53,6 +57,19 @@ if [ -z $cluster ] && [ -z $namespace ]; then
    usage
 fi
 
+if [ -z $secrets ]; then
+  secrets="hidden"
+fi
+
+if [ "$secrets" != "all" ] && [ "$secrets" != "off" ] && [ "$secrets" != "hidden" ]; then
+  echo "Unknown secrets verbosity level. Use one of 'off', 'hidden' or 'all'."
+  echo " 'all' - secret keys and data values are reported"
+  echo " 'hidden' - secrets with only data keys are reported"
+  echo " 'off' - secrets are not reported at all"
+  echo "Default value is 'hidden'"
```

Comment:
You're only telling the user what the default is if they use the option incorrectly. I think it should be part of the usage output.